packages/inter-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -350,7 +350,8 @@ const start = async (zcf, privateArgs, baggage) => {
     secondaryBrandToLiquidityMintProvider,
     () => {
       assert(reserveFacet, 'Must first resolveReserveFacet');
-      return E(reserveFacet).addIssuerFromAmm;
+      const reserveFacetP = E(reserveFacet);
+      return secondaryBrand => reserveFacetP.addIssuerFromAmm(secondaryBrand);
```

Comment:
why the const?
```suggestion
      return secondaryBrand => E(reserveFacet).addIssuerFromAmm(secondaryBrand);
```

Reply:
Because of types, in the calback TS think `reserveFacet` could have gone back to null. Since it's ok to build and cache the proxy, I'm keeping it in a const to satisfy types. A small deviation on my rule to not change runtime code to make types happy.

Reply:
A bare `E(expression)` not followed by a dot is disturbing. In the hypothetical world where we're using tildot, it would not even be possible to write it syntactically. I don't understand the typing problem --- but I usually don't. Would the tildot equivalent of @turadg 's suggested one liner have the same typing problem?

Reply:
I went for casting instead. As agreed, we shouldn't go through runtime hoops to satisfy the type system.
(Really wish we had non null assertions in JS or at least JSDoc)

Reply:
https://github.com/Agoric/agoric-sdk/pull/4422 got stalled out. Should I push it through?

Reply:
Yes, thanks!